daemon/cmd/kube_proxy_replacement.go

```diff
@@ -448,8 +448,7 @@ func finishKubeProxyReplacementInit(sysctl sysctl.Sysctl) error {
 
 	option.Config.NodePortNat46X64 = option.Config.IsDualStack() &&
 		option.Config.DatapathMode == datapathOption.DatapathModeLBOnly &&
-		option.Config.NodePortMode == option.NodePortModeSNAT &&
-		probes.HaveLargeInstructionLimit() == nil
```

Comment:
Should `probes.HaveLargeInstructionLimit()` be removed if it's no longer being called throughout the codebase?

Reply:
No. We still use the probe, but only in `CheckRequirements` in `pkg/datapath/linux/requirements.go`. Which is run during startup to assert that we have a proper kernel.